replica/table.cc

```diff
@@ -779,6 +781,7 @@ table::stop() {
     }
     return _async_gate.close().then([this] {
         return await_pending_ops().finally([this] {
+            _sstables_changed.broken();
```

Comment:
conditiona_variable::broken() doc:
    /// Signal to waiters that an error occurred.  \ref wait() will see
    /// an exceptional future<> containing the provided exception parameter.
    /// The future is made available immediately.

isn't it better to just call signal()?

Reply:
No, because the condition will not evaluate to true.
We'll need to check for the stop condition in the functor.
Breaking the condition variable works like semaphore::broken or gate::close in this sense

Reply:
Thanks for the explanation.